Sources/Internal/FrameProvider.swift

```diff
@@ -221,7 +226,7 @@ final class FrameProvider {
   {
     CGRect(
       x: layoutMargins.leading,
-      y: yContentOffset + daySize.height - separatorHeight,
+      y: layoutMargins.top + yContentOffset + daySize.height - separatorHeight,
```

Comment:
the pinned days of the week items also needed to use the top margin, otherwise it'll be obscured by the top nav bar.